tsconfig.json

```diff
@@ -8,8 +8,6 @@
     "noImplicitReturns": true,
     "noImplicitThis": true,
     "resolveJsonModule": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
```

Comment:
I would leave this checks, or there are some specific reason for removing it?

Reply:
Having unused variables does no harm, and in my experience it is very inconvenient to fix the code according to those checks every time I want to make some experimental  changes (e. g. commenting out some existing code). Moreover, TS still highlights unused variables (shows warnings). So it's just a matter of convenience for me. 
If it's only me who has this perception, I'll put them back 🙂.

Reply:
Will move this out of the scope.